quarkus.jdt/com.redhat.quarkus.jdt.test/src/main/java/com/redhat/quarkus/jdt/core/JDTQuarkusManagerDeploymentTest.java

```diff
@@ -45,10 +46,60 @@
  */
 public class JDTQuarkusManagerDeploymentTest {
 
+	static {

```

Comment:
prefer setting that in @BeforeClass and restoring the original level in @AfterClass, so other tests are not impacted

Reply:
Done with new class BaseJDTQuarkusManagerTest